### DISCUSSION TITLE
Add LG webOS TV app for Ente Photos

### DISCUSSION THREAD

**Discussion by JanOstrowka** [ente-io/ente#560] [Enhancements]
Dear Ente Team,

I would love to have a native LG webOS TV app for Ente Photos! Just like the Apple TV app that has been requested (https://github.com/ente-io/photos-app/issues/1448), having an app on LG TVs would allow me to easily view my Ente photos on the big screen. 

Browsing through all my favorite memories and photos on the TV would be an amazing experience. The LG webOS platform seems like a perfect fit for an Ente Photos app. The interface could be designed to take advantage of the TV display and remote control.

Features the LG webOS app could include:
- Browsing photos by date, album, or search
- Viewing photos in fullscreen slideshow mode
- Casting photos from mobile device to TV
- Caching photos for smooth performance
- Supporting 4K and high-res photos

I imagine the Ente Photos experience translating very well to LG TVs.

**Comment by vishnukvmd** [comment]
webOS supports packaging of web apps: https://webostv.developer.lge.com/develop/getting-started/web-app-types#hosted-web-app

It'd be interesting to see how we can reuse the existing code to deliver a good UX on large screens.

**Comment by JanOstrowka** [comment]
And if you are going to struggle to publish the app on LG Store, you can always publish here: https://github.com/webosbrew/
